backend/sky_lectures/settings/base.py

```diff
@@ -20,7 +20,7 @@
 # See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'zfxh=$!l0d)j2s4ym&mm$8oj5p@9rf=r#x4ni+78_0%t$1&9k5'
+SECRET_KEY = "zfxh=$!l0d)j2s4ym&mm$8oj5p@9rf=r#x4ni+78_0%t$1&9k5"
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
SECRET_KEY should be kept in environment variables. Try to move settings depending on the environment to the appropriate files.